deny.toml

```diff
@@ -35,18 +35,10 @@ wildcards = "deny"
 highlight = "all"
 # Certain crates/versions that will be skipped when doing duplicate detection.
 skip = [
-    { name = "cfg-if", version = "0.1" },               # from winit v0.26.0
-    { name = "core-foundation", version = "0.7" },      # from winit v0.26.0
-    { name = "core-foundation-sys", version = "0.7" },  # from winit v0.26.0
-    { name = "core-graphics", version = "0.19" },       # from winit v0.26.0
-    { name = "ndk", version = "0.5" },                  # from winit v0.26.1
-    { name = "ndk", version = "0.6" },                  # from rodio v0.16.0
-    { name = "ndk-glue", version = "0.5" },             # from winit v0.26.1
-    { name = "ndk-sys", version = "0.2" },              # from winit v0.26.1
     { name = "ndk-sys", version = "0.3" },              # from rodio v0.16.0
-    { name = "parking_lot", version = "0.11" },         # from winit v0.26.1
-    { name = "parking_lot_core", version = "0.8" },     # from winit v0.26.1
-    { name = "raw-window-handle", version = "0.4" },    # from wgpu v0.13.0
+    { name = "ndk", version = "0.7" },                  # from winit v0.27.4
+    { name = "ndk-glue", version = "0.7" },             # from winit v0.27.4
```

Comment:
This should skip the oldest version of those rather than the newest

Reply:
Done, thanks